<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="print_class_names">
            <property name="text">
             <string>Надрукувати імена класів</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="class1_print_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="class2_print_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="class3_print_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="class2_name">
            <property name="text">
             <string>Журнал</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_name_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_number_of_pages">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_size_of_page">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_number">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_period">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="class2_calc">
            <property name="text">
             <string>Обчислити</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="class2_res">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="class3_2">
              <property name="text">
               <string>Книга</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_name">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_number_of_pages">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_size_of_page">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_number">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_author">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="class3_calc">
              <property name="text">
               <string>Обчислити</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="class3_res">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="class2">
       <attribute name="title">
        <string>Magazine</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLineEdit" name="magazine_name">
            <property name="text">
             <string>Science</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="magazine_numbe_of_pages">
            <property name="text">
             <string>25</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="magazine_size_of_page">
            <property name="text">
             <string>0.3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="magazine_number">
            <property name="text">
             <string>100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="magazine_period">
            <property name="text">
             <string>28</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="magazine_set">
            <property name="text">
             <string>Встановити</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="class3">
       <attribute name="title">
        <string>Book</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLineEdit" name="book_name">
          <property name="text">
           <string>The Witcher</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="book_number_of_pages">
          <property name="text">
           <string>400</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="book_size_of_page">
          <property name="text">
           <string>0.25</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="book_number">
          <property name="text">
           <string>2000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="book_author">
          <property name="text">
           <string>A.Sapkovskiy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="book_set">
          <property name="text">
           <string>Встановити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
